packages/client/hmi-client/src/page/Home.vue

```diff
@@ -50,51 +50,39 @@
 				<TabView @tab-change="tabChange" :active-index="activeTabIndex" :key="activeTabIndex">
 					<TabPanel v-for="(tab, i) in TabTitles" :header="tab" :key="i">
 						<section class="filter-and-sort">
-							<div class="mr-3">
-								<tera-input-text
-									v-model="searchProjectsQuery"
-									placeholder="Search for projects"
-									id="searchProject"
-									:icon="isSearchLoading ? 'pi pi-spin pi-spinner' : 'pi pi-search'"
-									class="searchProjectsInput"
-								/>
-							</div>
-							<div>
-								<span v-if="view === ProjectsView.Cards">
-									<Dropdown v-model="selectedSort" :options="sortOptions" class="sort-options-dropdown" />
-								</span>
-								<MultiSelect
-									v-if="view === ProjectsView.Table"
-									:modelValue="selectedColumns"
-									:options="columns"
-									:maxSelectedLabels="1"
-									:selected-items-label="`{0} columns displayed`"
-									optionLabel="header"
-									@update:modelValue="onToggle"
-									placeholder="Add or remove columns"
-									class="p-inputtext-sm"
-								/>
-							</div>
-							<div>
-								<SelectButton
-									v-if="!isEmpty(searchedAndFilterProjects)"
-									:model-value="view"
-									@change="selectChange"
-									:options="viewOptions"
-									option-value="value"
-								>
-									<template #option="slotProps">
-										<span class="p-button-label">{{ slotProps.option.value }}</span>
-									</template>
-								</SelectButton>
+							<tera-input-text
+								v-model="searchProjectsQuery"
+								placeholder="Search for projects"
+								id="searchProject"
+								:icon="isSearchLoading ? 'pi pi-spin pi-spinner' : 'pi pi-search'"
+								class="searchProjectsInput"
+							/>
+							<Dropdown v-if="view === ProjectsView.Cards" v-model="selectedSort" :options="sortOptions" />
+							<MultiSelect
+								v-if="view === ProjectsView.Table"
+								:modelValue="selectedColumns"
+								:options="columns"
+								:maxSelectedLabels="1"
+								:selected-items-label="`{0} columns displayed`"
+								optionLabel="header"
+								@update:modelValue="onToggle"
+								placeholder="Add or remove columns"
+								class="p-inputtext-sm"
+							/>
+							<SelectButton :model-value="view" @change="selectChange" :options="viewOptions" option-value="value">
+								<template #option="slotProps">
+									<span class="p-button-label">{{ slotProps.option.value }}</span>
+								</template>
+							</SelectButton>
+							<aside>
```

Comment:
What are the improvements here? 
A one or two liner description would be nice, or conversely don't include people that are not aware of the motivation(s) of this change in the review :)